ios/ZingoMobile.xcodeproj/project.pbxproj

```diff
@@ -283,7 +283,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "set -e\n\nexport NODE_BINARY=/Users/juancarloscarmonacalvo/.nvm/versions/node/v16.16.0/bin/node\n../node_modules/react-native/scripts/react-native-xcode.sh\n";
+			shellScript = "set -e\n\nexport NODE_BINARY=/Users/juancarloscarmonacalvo/.nvm/versions/node/v16.20.2/bin/node\n../node_modules/react-native/scripts/react-native-xcode.sh\n";
```

Comment:
what is this for? why does it specify a path on your local machine? does that mean that this only works on your computer?

Reply:
You are right... mmmm, I'm going to try to change this in Macos, like `~/`... Thanks.

Reply:
done!